### PR TITLE
paig-examples added #52

### PR DESCRIPTION
**Pull request by adityasinght**
## Change Description

paig-examples added for quick reference

## Issue reference

This PR fixes issue #52 

## Checklist

- [x] I have reviewed the [contribution guidelines](https://github.com/privacera/paig/blob/main/docs/CONTRIBUTING.md)
- [x] My code includes unit tests
- [x] All unit tests and lint checks pass locally
- [x] My PR contains documentation updates / additions if required

### CODE REVIEW

File: paig-examples/python-openai-ai-app/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Simple AI Application demonstration using PAIG
+
+
+## Step 1: Start the PAIG server
+If you have not started the PAIG server, please follow the instructions in the [PAIG Quick Start Guide](https://docs.paig.ai/index.html) to start the PAIG server.
```
Comment (dbosco) [inline paig-examples/python-openai-ai-app/README.md:5]: We should have specific instructions for how start the PAIG server in standalone mode

File: paig-examples/python-openai-ai-app/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Step 2: Install the PAIG client and dependencies
+To install the PAIG client, run the following command:
+```bash
+pip install openai==1.11.1 paig-client
+```
+
+## Step 3: Download the PAIG configuration file
+Download the PAIG configuration file from the PAIG server and save it under directory 'privacera'.
```
Comment (dbosco) [inline paig-examples/python-openai-ai-app/README.md:14]: We might have to give link to the document to download the configuration

File: paig-examples/python-openai-ai-app/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+## Step 4: Set the OPENAI_API_KEY environment variable
+Set the OPENAI_API_KEY environment variable to your OpenAI API key.
+```bash
+export OPENAI_API_KEY=<your_openai_api_key>
+```
+
+## Step 5: Run the AI application
+Run the AI application using the following command:
+```bash
+python ai_app.py
```
Comment (dbosco) [inline paig-examples/python-openai-ai-app/README.md:28]: In most envs, e.g. in my mac, there is no `python`, it is `python3`. Should we standardize on this?

File: paig-examples/python-openai-ai-app/ai-app.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        updated_prompt_text = updated_prompt_text[0].response_text
+        print(f"User Prompt (After PAIG Shield): {updated_prompt_text}")
+        if prompt_text != updated_prompt_text:
+            print(f"Updated prompt text: {updated_prompt_text}")
+        # Call LLM with updated prompt text
+        PROMPT = f"""Use the following pieces of context to answer the question at the end.     
+      {updated_prompt_text}    
+      ANSWER:
+      """
+        response = openai_client.chat.completions.create(model="gpt-4", messages=[{"role": "user", "content": PROMPT}],
```
Comment (dbosco) [inline paig-examples/python-openai-ai-app/ai-app.py:36]: Can we make the model as `gpt-4o`? And create a global variable for `model`?

File: paig-examples/python-openai-ai-app/README.md (hunk 3 above)
Comment (pravin-bansod) [reply paig-examples/python-openai-ai-app/README.md:28]: we can start from creating virtual env - with that `python` would work and will be isolated setup.

**Review by amrakshay — APPROVED** [review]

**Review by pravin-bansod — APPROVED** [review]

**Pull request merged**